FakeNFT/Scenes /UserDetail/UserDetailModuleBuilder.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+class UserDetailModuleBuilder {
+    static func build(userId: String) -> UIViewController {
+        let networkClient = DefaultNetworkClient()
+        let interactor = UserDetailInteractor(networkClient: networkClient)
+        let router = UserDetailRouter()
+        let view = UserDetailViewController()
+        let presenter = UserDetailPresenter(view: view, interactor: interactor, router: router, userId: userId)
```

Comment:
Такие длинные строки порой тяжело воспринимать, можно так:

```suggestion
        let presenter = UserDetailPresenter(
            view: view, 
            interactor: interactor, 
            router: router, 
            userId: userId
        )
```